tests/sentry/tasks/test_invite_missing_org_members.py

```diff
@@ -75,6 +89,29 @@ def test_schedules_and_sends(self, mock_send_email, mock_send_notification):
 
         assert mock_send_notification.called
 
+    @with_feature("organizations:integrations-gh-invite")
+    @patch(
+        "sentry.notifications.notifications.missing_members_nudge.MissingMembersNudgeNotification.__init__",
+        return_value=None,
+    )
+    def test_email_content(self, mock_init_notification, mock_send_email, mock_send_notification):
```

Comment:
nit: test name clarity
```suggestion
    def test_excludes_filtered_emails(self, mock_init_notification, mock_send_email, mock_send_notification):
```